speechmatics/cli.py

```diff
@@ -364,6 +364,9 @@ def main(args=None):
     logging.basicConfig(level=get_log_level(args["verbose"]))
     LOGGER.info("Args: %s", args)
 
+    if args["command"] != "transcribe":
```

Comment:
This is why we need click 😎

Reply:
@viren-nadkarni can we resolve?

Reply:
I thought argparse only allows those subparsers which are actually registered?

Reply:
Yes, but there is the case, when you run `speechmatics` without arguments, this handles the case.

It could be rewritten to
```python
if args["command"] is None:
```

Reply:
https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_subparsers

`required=True`